app/partials/errorDetail-Q029.html

```diff
@@ -11,11 +11,11 @@
             <td class="col-group-first" aria-hidden="true">&nbsp;</td>
         </tr>
         <tr>
-            <th scope="col" class="id">LAR Number</th>
-            <th scope="col" class="state_code">State Code</th>
-            <th scope="col" class="county_code">County Code</th>
-            <th scope="col" class="census_tract">Census Tract</th>
-            <th scope="col" class="recommended col-group-first">MSA/MD</th>
+            <th scope="col" role="columnheader" class="id" ng-class="{'cf-icon icon-up': sortedBy === 'properties[\'LAR number\']' && !reverse, 'cf-icon icon-down': sortedBy === 'properties[\'LAR number\']' && reverse}" ng-click="sort('properties[\'LAR number\']')" ng-attr-aria-sort="{{isSortedBy('properties[\'LAR number\']')}}">LAR Number</th>
```

Comment:
According to http://stackoverflow.com/questions/24863531/using-aria-sort-in-validated-html5 and possibly http://www.w3.org/TR/wai-aria/states_and_properties#aria-sort but it's harder for me to interpret, `role="columnheader"` is needed despite `scope="col"`.

I can move it to the controller but wasn't sure how much of an advantage it would gain.

Guess I didn't need the `ng-attr-`, whoops.